qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -18,6 +27,11 @@
             nic_model_nic1 = virtio
             nic_model_nic2 = rtl8139
             device_name = "Red Hat VirtIO Ethernet Adapter"
+            variants:
```

Comment:
Is it better to move this variant out? 
variants:
      - @default:
      - with_floopy:
         xxxx
variants:
      - with_netkvm:
         xxxx
      - with_viorng:
         no with_floppy
         xxxx

Reply:
Thanks for your advice, has updated.